content/cybersecurity/concepts/spear-phishing/spear-phishing.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Spear Phishing'
+Description: 'Spear phishing is a type of social engineering scam that attempts to trick the recipient into providing confidential information to the attacker.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Validation'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Spear phishing** is a type of social engineering scam that attempts to trick the recipient into providing confidential information to the attacker. Spear phishing is a more targeted cyber attack than phishing.
```

Comment:
Good op to link out to cyber attack entry: https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack